components/org.wso2.carbon.identity.oauth.uma.resource.service/src/main/java/org/wso2/carbon/identity/oauth/uma/resource/service/ResourceService.java

```diff
@@ -30,15 +30,15 @@
 
     boolean deleteResource(String resourceId) throws UMAServerException, UMAClientException;
 
-    List<String> getResourceIds(String resourceOwnerName, String consumerKey) throws UMAServerException
-            , UMAClientException;
+    List<String> getResourceIds(String resourceOwnerName, String userDomain, String consumerKey) throws
```

Comment:
Add method comments to all methods.

Reply:
fixed: https://github.com/wso2-extensions/identity-oauth-uma/pull/68